### ISSUE TITLE
Add history tab in the sidebar

### ISSUE THREAD

**Issue by benweet** [classeur/classeur#17]
A history tab would be useful to track changes in a file, open an old revision, or rollback some edits.

**Comment by johnaxel** [comment]
Hi @benweet -- I saw you committed some code for the File History tab but I don't see it in the UI. Is it released yet? I'm happy to act as a beta tester at some point if that's useful ;)

https://github.com/classeur/classeur/commit/b9321c6515ad5327d84ba7c516debcfd859427c0

Love Classeur so far and can't wait till this feature is out.

**Comment by vagelim** [comment]
@johnaxel In a document, navigate to the _Menu_ tab, enable _Side bar_ slider (should turn blue). In the side bar that pops out, click the icon resembling Time Machine. (see image)
[![History pane](http://i.imgur.com/Ane7C0j.png)](http://i.imgur.com/Ane7C0j.png)

**Comment by johnaxel** [comment]
:+1: thanks!

**Comment by ghost** [comment]
Looks like still not working!

**Comment by orschiro** [comment]
Dear @winteum,

Can you confirm that you have an empty history sidebar?

~Robert

![image](https://cloud.githubusercontent.com/assets/170947/16630759/12837742-43bc-11e6-9964-4ccb72e67174.png)

**Comment by ghost** [comment]
Yes, empty!

**Comment by benweet** [comment]
Hi @winteum
Can you confirm you're signed in?

**Comment by ghost** [comment]
Yes, through my Google account.

**Comment by benweet** [comment]
Usually history appears after you modify the files multiple times with a couple of minutes of interval. Can you try with an old file you've been working on?

**Comment by ghost** [comment]
Yes, I was trying this. I'll continue to try modifying to see if it's shows there.

**Comment by benweet** [comment]
Alright. A snapshot of the file should take place after 2-3 minutes of
inactivity.

**Comment by ghost** [comment]
I even created another file but still no snapshots neither.

**Comment by benweet** [comment]
We can reproduce the issue. Investigating...

**Comment by benweet** [comment]
@winteum should be fixed now. Can you confirm?

**Comment by ghost** [comment]
Yes, it's working!
